data/maps/Route11_EastEntrance_1F/scripts.pory

```diff
@@ -6,7 +6,33 @@ Route11_EastEntrance_1F_EventScript_TopGuard::
 	msgbox Route11_EastEntrance_1F_Text_ManInLavenderRatesNames, MSGBOX_NPC
 	end
 
-Route11_EastEntrance_1F_EventScript_BottomGuard::
-	msgbox Route11_EastEntrance_1F_Text_RockTunnelToReachLavender, MSGBOX_NPC
-	end
+
 `
+script Route11_LosselleDad {
+	lock
+	faceplayer
+	if(!flag(FLAG_CSR_LOSSELLE_DAD) || !flag(FLAG_RESCUED_LOSTELLE)) {
```

Comment:
Is the first half of this condition necessary? 🤔 doesn't really seem like it does anything rn

Reply:
It makes it so that the first time you talk to him no matter what, you'll get the "Oh where's Losselle" line, but every time after that it only displays if you haven't rescued her ^^

Reply:
Basically there's four ways the convo can go:
First time talking to dad + haven't rescued losselle
first time talking to dad + have rescued losselle
Second time talking to dad + haven't rescued losselle
Second time talking to dad + have rescued losselle

1 and 3 are the same situation and only display the first dialogue; 2 will display both dialogues and 4 will only display the second dialogue ^^